Production_scripts/B05a_Process_A&E_(AE2)_extract.R

```diff
@@ -0,0 +1,1094 @@
+#####################################################
+# A&E Extract
+# Author: Catherine Holland
+# Date: February 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input -
+# Description - Process A & E extract
+#####################################################
+
+
+library(dplyr)
+library(tidyr)
+library(ggplot2)
+library(createslf)
+library(slfhelper)
+
+
+## get data ##
+
+# latest year
+latest_year <- 1920
+
+# function here till PR pushed
+get_year_dir <- function(year, extracts_dir = FALSE) {
+  year_dir <- fs::path("/conf/sourcedev/Source_Linkage_File_Updates", year)
+
+  year_extracts_dir <- fs::path(year_dir, "Extracts")
+
+  return(dplyr::if_else(extracts_dir, year_extracts_dir, year_dir))
+}
+
+
+ae_episode_extract <- readr::read_csv(
+  paste0(
+    get_year_dir(year = latest_year),
+    "/Extracts/A&E-episode-level-extract-20",
+    latest_year,
+    ".csv.gz"
+  )
+) %>%
+  # rename
+  rename(
+    record_keydate1 = "Arrival Date",
+    record_keydate2 = "DAT Date",
+    dob = "Pat Date Of Birth [C]",
+    postcode_epi = "Postcode (epi) [C]",
+    postcode_chi = "Postcode (CHI) [C]",
+    age = "Age at Midpoint of Financial Year",
+    ae_alcohol = "Alcohol Involved Code",
+    alcohol_adm = "Alcohol Related Admission",
+    ae_arrivalmode = "Arrival Mode Code",
+    keyTime1 = "Arrival Time",
+    ae_attendcat = "Attendance Category Code",
+    ae_bodyloc = "Bodily Location Of Injury Code",
+    lca = "Council Area Code",
+    ae_disdest = "Discharge Destination Code",
+    keyTime2 = "DAT Time",
+    diag1 = "Disease 1 Code",
+    diag2 = "Disease 2 Code",
+    diag3 = "Disease 3 Code",
+    falls_adm = "Falls Related Admission",
+    gpprac = "GP Practice Code",
+    hscp = "HSCP of Residence Code - current",
+    hbrescode = "NHS Board of Residence Code - current",
+    hbtreatcode = "Treatment NHS Board Code - current",
+    chi = "Pat UPI [C]",
+    gender = "Pat Gender Code",
+    ae_patflow = "Patient Flow Code",
+    ae_placeinc = "Place of Incident Code",
+    ae_reasonwait = "Reason for Wait Code",
+    refsource = "Referral Source Code",
+    selfharm_adm = "Self Harm Related Admission",
+    submis_adm = "Substance Misuse Related Admission",
+    sigfac = "Significant Facility Code",
+    cost_total_net = "Total Net Costs",
+    location = "Treatment Location Code",
+    case_ref_number = "Case Reference Number"
+  ) %>%
+  # date types
+  mutate(
+    dob = as.Date(dob),
+    record_keydate1 = as.Date(record_keydate1),
+    record_keydate2 = as.Date(record_keydate2)
+  )
```

Comment:
I've added in the boxi_extract_path function and while doing this i've added in using `col_type` which allows you to specify the formats. I've removed this